installing/installing_vsphere/uninstalling-cluster-vsphere-installer-provisioned.adoc

```diff
@@ -8,4 +8,9 @@ toc::[]
 
 You can remove a cluster that you deployed in your VMware vSphere instance by using installer-provisioned infrastructure.
 
+[NOTE]
+====
+Using `openshift-install destroy cluster`, vSphere volumes are not automatically deleted during {product-title} uninstallation. Cluster administrator must manually find vSphere volumes and delete them.
```

Comment:
```suggestion
When you run the `openshift-install destroy cluster` command to uninstall {product-title}, vSphere volumes are not automatically deleted. The cluster administrator must manually find the vSphere volumes and delete them.
```

Reply:
@kalexand-rh Fixed. Thanks for the suggestion.